setup.py

```diff
@@ -16,10 +16,13 @@
         "argcomplete",
         "docutils",
         "humanize",
+        "pycocotools",
         "pyyaml>=5.1",
         "requests",
         "sh",
         "tqdm",
+        "torch",
+        "torchvision",
```

Comment:
Actually I don't think we should add torch/torchvision here.
My reasoning is that you could use our cli without ever needing torch installed, so why force it.
I also want use to add support for tensorflow (darwin.tf) in the future, but we should force the user to install both tf and pytorch.

It's better if we add a check in darwin/torch/__init__.py that complains if torch is not available. 
Which is also a reason not to include torch in the darwin/__init__.py

Reply:
Agreed